### DISCUSSION TITLE
Make border stroke dashed

### DISCUSSION THREAD

**Discussion by DasLixou** [AvaloniaUI/Avalonia#8380] [Q&A] [answered]
How can i make my BorderStroke dashed? I use ```<Border Margin="4" BorderBrush="#FAFAFA" CornerRadius="11" BorderThickness="1" BorderDashOffset="5">``` but it's still a complete line without gaps

**Comment by MarkusKgit** [comment, marked as answer]
`BorderDashArray="5,5"`

**Comment by 29654761** [comment]
It will not work while border one side BorderThickness="0 0 0 1" 
BorderDashArray not works

**Reply by timunie** [reply]
dashed border was only a preview feature. However, it pointed out that this feature has too many draw backs and thus it was removed for final release. Sorry. #11548
